transport.rst

```diff
@@ -71,17 +84,15 @@ Matrix allows to check for the presence of a user. Clients should listen for cha
 Sending transfer messages to other nodes
 ----------------------------------------
 
-Direct Message, which is modeled as a room with 2 participants.
-Channel rooms have the name in the format ``raiden_<network_name_or_id>_<peerA>_<peerB>``, where ``peerA`` and ``peerB`` are sorted in lexical-order. As the users may roam to other homeservers, participants should keep listening for user-join events in the presence room, if it's a user of interest (with which it shares a room), with valid signed ``displayName`` and not yet in the room we share with it, invite it to the room.
 
+Private channel rooms random room_ids in the format ``random_room_id:homeserver``, where the
```

Comment:
```suggestion
Private channel rooms have random room ids in the format ``random_room_id:homeserver``, where the
```

We could consider the random ids as an implementation detail not relevant for the spec, since (afaik) only the convenience room ids mentioned below are used?

Reply:
No it's the other way around, we only use random room_ids and give the rooms aliases afterwards